dev/Repeater/APITests/FlowLayoutTests.cs

```diff
@@ -146,6 +146,15 @@ public void ValidateGridLayout()
                         ((UniformGridLayout)panel.Layout).MinColumnSpacing = minColumnSpacing;
                         Content.UpdateLayout();
                         ValidateGridLayoutChildrenLayoutBounds(om, (i) => panel.Children[i], itemMinorSize, itemMajorSize, minRowSpacing, minColumnSpacing, panel.Children.Count, panel.DesiredSize);
+
+                        minRowSpacing = 5;
+                        minColumnSpacing = 10;
+                        ((UniformGridLayout)panel.Layout).MinRowSpacing = minRowSpacing;
```

Comment:
If this were product code I'd be asking you to fetch and dynamically cast that once. Reduce reuse recycle. And I try to hold test code to product code standards as much as possible. So... I'm guess I'm saying fix this too? :)